### PR TITLE
Manage private and public links on the same page.

### PR DESCRIPTION
**Pull request by difanyang**
Looks like this:
image
image

### CODE REVIEW

**Review by jiaxi312 — APPROVED** [review]

**Comment by xiaoluLucy814** [comment]
Just a general comment, I guess we don't need to display "go public" for a public link or "go private" for a private link

File: easylinks/src/main/java/com/google/step/servlets/ManageServlet.java [hunk 1]
```diff
@@ -138,6 +140,7 @@ private static void goPublic(HttpServletRequest request, HttpServletResponse res
         response.getWriter().println("Repeated shortcut.");
         return;
       }
+      easyLinkEntity.setProperty("status", "Public");
       easyLinkEntity.setProperty("email", ServletHelper.ADMIN);
       easyLinkEntity.setProperty("creator", ServletHelper.USERSERVICE.getCurrentUser().getEmail());
```
Comment (xiaoluLucy814) [inline easylinks/src/main/java/com/google/step/servlets/ManageServlet.java:145]: don't need to setProperty for creator as it stays same

**Comment by difanyang** [comment]
> Just a general comment, I guess we don't need to display "go public" for a public link or "go private" for a private link

You are right. But the frontend would not be able to tell whether a link is public or private.

**Comment by xiaoluLucy814** [comment]
> > Just a general comment, I guess we don't need to display "go public" for a public link or "go private" for a private link
> 
> You are right. But the frontend would not be able to tell whether a link is public or private.

What about the status attribute?

**Comment by difanyang** [comment]
> > > Just a general comment, I guess we don't need to display "go public" for a public link or "go private" for a private link
> > 
> > 
> > You are right. But the frontend would not be able to tell whether a link is public or private.
> 
> What about the status attribute?

I'll try that. Thanks for the idea!

**Comment by difanyang** [comment]
The UI:
image

**Review by xiaoluLucy814 — APPROVED** [review]

**Pull request merged**